frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const Navbar = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <Nav>
+            <Logo href="">
```

Comment:
Da para já adicionar o `href` certo aqui para sempre mandar o usuário para o home:
```suggestion
            <Logo href="/">
```